app/controllers/Auth.scala

```diff
@@ -190,16 +199,21 @@ class Auth @Inject() (
     })
   }
 
+
   def authenticate = Action.async { implicit request =>
     signInForm.bindFromRequest.fold(
       bogusForm => Future.successful(BadRequest(views.html.auth.signIn(bogusForm, socialProviderRegistry))),
       signInData => {
         val credentials = Credentials(signInData.email, signInData.password)
-        credentialsProvider.authenticate(credentials).flatMap { loginInfo => 
+        pool1Service.pool1user(signInData.email).flatMap {
+          case Some(pooluser) if !pooluser.confirmed =>
```

Comment:
What does `!pooluser.confirmed` mean? Check that the user is confirmed in Pool 1?